src/main/java/it/gov/pagopa/pu/workflow/impl/PaymentsReportingIngestionWFImpl.java

```diff
@@ -0,0 +1,99 @@
+package it.gov.pagopa.pu.workflow.impl;
```

Comment:
I would take interface and implementation on the same package.
Instead I would create a package based on an aggregation logic:

workflow.ingestionflow -> here all the ingestionFlowFile related WF (PaymentsReporting, Treasury, ...)
workflow.classify -> here all the WF related to classification